package.json

```diff
@@ -6,9 +6,9 @@
     "module": "./dist/webview-bridge.mjs",
     "types": "./dist/index.d.ts",
     "scripts": {
-        "build": "del dist && rollup -c && uglifyjs --compress --mangle -o dist/webview-bridge-umd.min.js dist/webview-bridge-umd.js && uglifyjs --compress --mangle -o dist/webview-bridge-iife.min.js dist/webview-bridge-iife.js && cpx ./src/webview-bridge-cjs.js.flow ./dist",
+        "build": "del dist && rollup -c && uglifyjs --compress --mangle -o dist/webview-bridge-umd.min.js dist/webview-bridge-umd.js && uglifyjs --compress --mangle -o dist/webview-bridge-iife.min.js dist/webview-bridge-iife.js && cp ./src/webview-bridge-cjs.js.flow ./dist",
         "test": "jest",
-        "lint": "tslint *.ts",
+        "lint": "eslint src/*.ts",
```

Comment:
does this glob include subfolders too?

Reply:
hmm, I don't think so. Changing to src/**/*.ts